src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -73,6 +73,12 @@ export const cPlusPlusOptions = {
         "not-permissive",
         "secondary"
     ),
+    constStyle : new EnumOption(
```

Comment:
If the value of this is a `boolean`, then let's please call it `westConst`.

Reply:
My actual intention was to force not the have a bool, such as `--west-cost`, but a named option, `--const-style west-const` (which would be the default if not specified to be backwards compatible) or `--const-style east-const`.

Reply:
I mean the name of the TypeScript property, not the name of the command line option.

If I read `if (this._options.constStyle)` I interpret it as "do we have a const style?".  And even if I know that it distinguishes between west and east, I don't know whether `true` is west or east.  Whereas `if (this._options.westConst)` answers those questions a lot better.

Reply:
I see that some of the other boolean properties don't follow this convention either:

- `typeSourceStyle` with options `single-source` and `multi-source`; should be called `singleSource`
- `includeLocation` with options `local-include` and `global-include`; should be called `localInclude`
- `codeFormat` with options `with-struct` and `with-getter-setter`; should be called `withGetterSetter`

I'm not going to touch that but I'm saying there isn't a consistent style here.

Reply:
I agree. I'm not happy about that, either.